tests/openvino/native/test_node_utils.py

```diff
@@ -86,3 +88,24 @@ def test_get_weight_channel_axes_for_matmul(weights_port_id, transpose, shape, e
 
     assert len(actual_channel_axes) == len(expected_channel_axes)
     assert all(a == b for a, b in zip(actual_channel_axes, expected_channel_axes))
+
+
+@pytest.mark.parametrize(
+    "metatype, inputs_attributes, expected_channel_axis",
+    [
+        (OVMatMulMetatype, {"transpose": False}, -1),
+        (OVMatMulMetatype, {"transpose": True}, 1),
+        (OVConvolutionMetatype, {}, 1),
+    ],
+)
+def test_get_activation_channel_axis(metatype, inputs_attributes, expected_channel_axis):
```

Comment:
Could you explain `get_activation_channel_axis(...)` returns the input axis number of the input or output activation tensor which correspond to it channel?

Reply:
Updated. For now, it returns the channel axis for activation (aka input).